### ISSUE TITLE
java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "getdtablesize"

### ISSUE THREAD

**Issue by LeroyChristopherDunn** [gluonhq/client-maven-plugin#236]
Hi there 

My client run task with android target fails with the below link error:

`
 E/AndroidRuntime(19668): java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "getdtablesize" referenced by "/data/app/hello.hellodesktop-XiRilnh49ahx5XXKtJJqtg==/lib/arm64/libsubstrate.so"...
`

This specifically occurs when launching our enterprise jar with [Assisted Configuration of Native Image Builds](https://github.com/oracle/graal/blob/master/substratevm/CONFIGURE.md). I'm unfortunately unable to replicate the issue in a simple project

Env:
GraalVM CE 20.3.0-dev-20200814_0227 and it's bundled Java 11
Ubuntu 16.04  
Maven client plugin version 0.1.31

[full-stacktrace.log](https://github.com/gluonhq/client-maven-plugin/files/5103886/full-stacktrace.log)

Any assistance would be greatly appreciated

**Comment by jperedadnr** [comment]
Make sure you run with latest release GraalVM 20.2.0 (the log shows /home/leroy/Programs/graalvm-ce-java11-20.2.0-dev) from https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-20.2.0

For some reason, one of your dependencies is adding the `getdtablesize` symbol, but there is no implementation provided and that causes the runtime failure. Are you including native libraries in your project?

**Comment by LeroyChristopherDunn** [comment]
I took the following steps to resolve the issue:

1. I disabled log4j JMX with a system property as per [log4j documentation](https://logging.apache.org/log4j/log4j-2.3/manual/jmx.html)

2. We were using async loggers in our project. I made sure to disable them by removing the async logging system property and the Disruptor dependency as per [log4j documentation](https://logging.apache.org/log4j/2.x/manual/async.html)

3. client:build was now failing with a ClassNotFound error. I explicitly added back the now unused Disruptor dependency as per [this issue](https://github.com/oracle/graal/issues/671)
